### ISSUE TITLE
Add default value of `assets_dir`

### ISSUE THREAD

**Issue by surajssd** [kinvolk/lokomotive#709]
Make `asset_dir` an optional variable by adding default value as `assets`.

This will be one less variable user has to provide when starting out.

**Comment by iaguis** [comment]
This sounds OK to me. Tagging @johananl for UX opinions :)

**Comment by johananl** [comment]
Yes! :clap: I think we can default to `.assets` (hidden) and ensure we bail out if `lokoctl` is executed in a directory which doesn't contain a `*.lokocfg` file to avoid accidentally overwriting an unrelated directory named `.assets`. We could also decide to be extra safe and set the default to `.lokoctl_assets`.

Another note - if we make `asset_dir` transparent to the user, we should consider removing the "you can delete the assets dir now" message.

**Comment by invidian** [comment]
>Another note - if we make asset_dir transparent to the user, we should consider removing the "you can delete the assets dir now" message.

But this will clutter the disk space, as asset dir is not small :/

>Yes! clap I think we can default to .assets (hidden) and ensure we bail out if lokoctl is executed in a directory which doesn't contain a *.lokocfg file to avoid accidentally overwriting an unrelated directory named .assets. We could also decide to be extra safe and set the default to .lokoctl_assets.

`.lokoctl_assets` as default value seems reasonable to me.

However, assuming that `*.lokocfg` files are versioned using Git, it would be good to put asset directory outside the current directory. We could use for example `$XDG_DATA_HOME/lokoctl/<cluster name>` (so by default `$HOME/.local/share/lokoctl/<cluster name>`). But maybe that's too complicated for now.
